src/NServiceBus.RavenDB.Tests/RavenDBPersistenceTestBase.cs

```diff
@@ -11,6 +12,7 @@ public class RavenDBPersistenceTestBase : RavenTestBase
         [SetUp]
         public virtual void SetUp()
         {
+            Console.WriteLine($"Execution Environment: 64-bit OS:{Environment.Is64BitOperatingSystem}, 64-bit Process:{Environment.Is64BitProcess}");
```

Comment:
Earlier debugging line when the build server and Voron was hating on me. I will remove.